### DISCUSSION TITLE
Github/Gitlab pull request integration

### DISCUSSION THREAD

**Discussion by sverch** [planetscale/discussion#38] [Feature Requests] [answered]
I'd like to be able to use the branching features without having to ever interact with Planetscale directly, by setting up some configuration on my source repository that automatically manages it as developers go through their normal workflow, forking databases whenever there's a new pull request on the original repo that includes a schema change.

It looks like you already support [service tokens](https://docs.planetscale.com/reference/planetscale-cli) so it may already be possible to build that externally.

**Comment by hallaroo** [comment, marked as answer]
Hi @sverch! Thanks so much for the suggestion. A pull request integration is indeed on the roadmap and I've filed an issue with the team just to track of this request. We'll keep this updated as we get more information. Thanks again!
